scripts/validate-other-market-data.ts

```diff
@@ -389,10 +412,10 @@ async function validateAgainstLocalnet(proposals: Proposal[]): Promise<void> {
             stepBaseQuantums: Long.fromNumber(proposal.params.stepBaseQuantums),
             subticksPerTick: proposal.params.subticksPerTick,
             delayBlocks: proposal.params.delayBlocks,
-            // marketType:
-            //   proposal.params.marketType === 'PERPETUAL_MARKET_TYPE_ISOLATED'
-            //     ? PerpetualMarketType.PERPETUAL_MARKET_TYPE_ISOLATED
-            //     : PerpetualMarketType.PERPETUAL_MARKET_TYPE_CROSS,
+            marketType:
+              proposal.params.marketType === 'PERPETUAL_MARKET_TYPE_ISOLATED'
+                ? PerpetualMarketType.PERPETUAL_MARKET_TYPE_ISOLATED
+                : PerpetualMarketType.PERPETUAL_MARKET_TYPE_CROSS,
```

Comment:
@jaredvu are we ready to uncomment this?

Reply:
We aren't totally ready to uncomment this yet. We want to make sure the json is compatible with v4.0